### PR TITLE
Fixed deindex of medias

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | fixes https://github.com/sulu/sulu/issues/3493
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR adds the remove call for `FileVersionMeta` - this will trigger the `onRemove` event of doctrine to deindex the media from massive-search index.

### CODE REVIEW

**Comment by danrot** [comment]
Tests are failing. And it is new to me that we put an empty line before `break`.

**Comment by adsc-cloudtec** [comment]
I've tested this fix in a local sulu-standard installation and it works.

**Comment by wachterjohannes** [comment]
@danrot the tests should go green now :)

**Comment by wachterjohannes** [comment]
but the line breaks before the switch-break will be forced by styleci. should we try to turn that of?

**Comment by danrot** [comment]
The [Symfony Coding standards] force new lines only before the `return` statement, therefore I would try to switch it off. But we should probably include the entire @sulu/core-team into the decision.

File: src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php [hunk 1]
```diff
@@ -355,6 +355,7 @@ private function modifyMedia($uploadedFile, $data, $user)
             /** @var FileVersion $fileVersion */
             if ($version == $fileVersion->getVersion()) {
                 $currentFileVersion = $fileVersion;
+
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php]: Remove also this blank line.

**Pull request merged**